doc/source/api-key-creation.rst

```diff
@@ -4,7 +4,7 @@
 API Key Creation
 ====================
 
-API keys provide a flexible mechanism for granting access to your account and are used to authenticate requests made using the Conduce REST API.  They can be used in place of username/password authentication.  You may create as many API keys as you need, and may wish to use different API keys for performing specific API operations.  API keys can also be used to grant other users temporary access to your account.
+API keys are used to authenticate requests made using the Conduce REST API.  They can be used in place of username/password authentication.  You may create as many API keys as you need, and you may wish to use different API keys for performing specific API operations.  API keys can also be used to grant other users temporary access to your account.
```

Comment:
I like the first edit, but the addition of "you" is not necessary.

Reply:
Actually, what prompted you to remove "provide a flexible mechanism for granting access to your account and "?

Reply:
you're right. Is that comma right before that necessary? It sounded a bit strange to me

Reply:
It seemed like filler/fluff. Isn't that what any api key does? 

Though actually, 1 thing I have started to see as specific to our api keys is that they substitute for a user, their password, and a specific environment. Usually I think of an API key as just specific to a user(with a password).

So maybe that could be put back in as something like: API keys provide a flexible mechanism for granting access to your account on a specific conduce environment.

What do you think?